src/components/navBar/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import style from '@/components/navBar/navBar.module.scss';
+
+const NavBar = () => (
+  <div>
+    <nav className={style.navBar}>
+      <ul>
+        <li>
+          <a style={{ padding: '7px' }} href="https://realdevsquad.com">
+            <img
+              src="/RDSLogo.png"
```

Comment:
Create a variable for the image path, place it outside of the components,
`const RDSLogo = '/RDSLogo.png';`